### PR TITLE
improve wording

### PR DESCRIPTION
**Pull request by Findus23**
Requested in https://www.transifex.com/matomo/matomo/translate/#de/$/317268044?q=issue%3Aopen

It's still not ideal, but a lot more readable I think

### CODE REVIEW

**Comment by diosmosis** [comment]
Definitely a bit more readable! The UI tests need to be updated, but otherwise it's good to merge.

**Comment by ujdhesa** [comment]
Here’s another attempt:

“Enter 0 to allow unlimited actions (default). Under normal circumstances, most sites never experience more than around 100 to 300 actions per visit. Therefore, in many cases it might be safe to assume that if something has caused more actions than the configured amount of actions, it might be actually a tracking spam, a bot or something else unusually causing these actions. Matomo will in this case stop recording further actions for that visit, in order to have less inaccurate data and to reduce server load. The IP address of this visit will then be blocked for up to 24 hours.”

”Something”, since it could be “a bot, or something else”
“Unnaturally” seems to me inappropriate for something caused by man (part of nature) through software and/or hardware.

**Comment by tsteur** [comment]
Looks like we need to resolve some merge conflicts here and potentially maybe update screenshot tests

**Comment by AltamashShaikh** [comment]
> Here’s another attempt:
> 
> “Enter 0 to allow unlimited actions (default). Under normal circumstances, most sites never experience more than around 100 to 300 actions per visit. Therefore, in many cases it might be safe to assume that if something has caused more actions than the configured amount of actions, it might be actually a tracking spam, a bot or something else unusually causing these actions. Matomo will in this case stop recording further actions for that visit, in order to have less inaccurate data and to reduce server load. The IP address of this visit will then be blocked for up to 24 hours.”
> 
> ”Something”, since it could be “a bot, or something else” “Unnaturally” seems to me inappropriate for something caused by man (part of nature) through software and/or hardware.

@Findus23 I think we should remove “Unnaturally” as suggested

File: lang/en.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+        "SettingMaxActionsDescription": "Enter 0 to allow unlimited actions (default). Most sites never have more than around 100 to 300 actions within one visit under normal circumstances. In many cases it might therefore be safe to assume that if someone has caused more actions than the configured amount of actions, it might be actually tracking spam or a bot or something else like non-human activity is causing these actions. Matomo will in this case stop recording further actions for that visit to have less inaccurate data and to reduce server load. The IP address of this visit will then be blocked for up to 24 hours.",
```
Comment (AltamashShaikh) [inline lang/en.json:4]: @tsteur I updated the wording here previously it had a word called "unnatural"

**Pull request merged**